packages/xchain-cosmos/src/cosmos/sdk-client.ts

```diff
@@ -224,29 +225,73 @@ export class CosmosSDKClient {
     return this.signAndBroadcast(unsignedStdTx, privkey, AccAddress.fromBech32(from))
   }
 
-  async sign(unsignedStdTx: StdTx, privkey: PrivKey, signer: AccAddress): Promise<StdTx> {
+  async signAndBroadcast(unsignedStdTx: StdTx, privkey: PrivKey, signer: AccAddress): Promise<BroadcastTxCommitResult> {
     this.setPrefix()
 
     let account: BaseAccount = (await auth.accountsAddressGet(this.sdk, signer)).data.result
     if (account.account_number === undefined) {
       account = BaseAccount.fromJSON((account as BaseAccountResponse).value)
     }
 
-    return auth.signStdTx(
+    const signedStdTx = auth.signStdTx(
       this.sdk,
       privkey,
       unsignedStdTx,
       account.account_number.toString(),
       account.sequence.toString(),
     )
-  }
 
-  async broadcast(signedStdTx: StdTx): Promise<BroadcastTxCommitResult> {
     return (await auth.txsPost(this.sdk, signedStdTx, 'sync')).data
   }
 
-  async signAndBroadcast(unsignedStdTx: StdTx, privkey: PrivKey, signer: AccAddress): Promise<BroadcastTxCommitResult> {
-    const signedTx = await this.sign(unsignedStdTx, privkey, signer)
-    return await this.broadcast(signedTx)
+  async transferSigned({
```

Comment:
Maybe rename to `transferSignedOffline`? Pls add a comment what this method is about.